src/components/Avatar/Avatar.scss

```diff
@@ -38,27 +38,28 @@ $large-size: rem(60px);
 }
 
 .styleOne {
-  background: color('teal');
+  color: var(--p-warning-text, color('white'));
```

Comment:
Having success and warning roles used in avatars feels wrong to me, highlight a bit less. But maybe this is a time when it is okay to have an exception to color role usage?

Reply:
Otherwise I wonder if it’s possible, to @alex-page’s point, to take a single decorative color and generate a range of complimentary colors that actually look good regardless of the base color. I think that logic would be based on reverse engineering our use of the color.